### PR TITLE
Fix filter species on inconsistent species identification

### LINKED ISSUE #83: test filter_species with combine_subspecies
Currently, `filter_species` throws out anything with `unid.` in the name, on the theory that it can't be identified to species.  But the following three species (and possibly some others?) are identified to species, just not to subspecies.  I think that means we're throwing out observations for them incorrectly.

```
1    (unid. Red/Yellow Shafted) Northern Flicker
2                   (unid. race) Dark-eyed Junco
3 (unid. Myrtle/Audubon's) Yellow-rumped Warbler
```

### PR DESCRIPTION
**Pull request by sdtaylor**
fixes #83

### CODE REVIEW

File: R/forecast-bbs-core.R [hunk 1]
```diff
@@ -60,7 +60,8 @@ db_engine=function(action, db='./data/bbsforecasting.sqlite', sql_query=NULL,
 #' Removes waterbirds, shorebirds, owls, kingfishers, knightjars,
 #' dippers. These species are poorly sampled due to their aquatic or
 #' noctural nature. Also removes taxa that were either partially unidentified
-#' (e.g. "sp.") or were considered hybrids (e.g. "A x B").
+#' (e.g. "sp.") or were considered hybrids (e.g. "A x B") or were listed as more
+#' than one species (e.g. "B / B")
```
Comment (davharris) [inline R/forecast-bbs-core.R]: Should this be "A / B"?

Comment (sdtaylor) [inline R/forecast-bbs-core.R]: Yes

**Comment by sdtaylor** [comment]
pinging @davharris

**Comment by davharris** [comment]
👍

**Pull request merged**